fixtures/thoth_thamos_advise_finished_1.json

```diff
@@ -0,0 +1,11 @@
+{
+    "action": "finished",
+    "installation": {
+        "id": 6181026
+    },
+    "repo_url": "https://api.github.com/repos/thoth-station/Qeb-Hwt",
```

Comment:
I think this fixture can be removed, as the `repo_url` provided is not the right one :)